From 773f04bc73ee862afb1cb79e8dbbf4e4aa54f8b6 Mon Sep 17 00:00:00 2001
From: Carlos Garcia Campos <cgarcia@igalia.com>
Date: Mon, 30 Jan 2023 03:29:09 -0800
Subject: [PATCH] Unreviewed. [GTK] Do not include WebKitPrintCustomWidget.h in
 webkit.h for GTK4

It was removed from GTK4 build in 259286@main.

* Source/WebKit/UIProcess/API/glib/webkit.h.in:

Canonical link: https://commits.webkit.org/259551@main
---
 Source/WebKit/UIProcess/API/glib/webkit.h.in | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/Source/WebKit/UIProcess/API/glib/webkit.h.in b/Source/WebKit/UIProcess/API/glib/webkit.h.in
index 358e65c095be..fa00e623ba89 100644
--- a/Source/WebKit/UIProcess/API/glib/webkit.h.in
+++ b/Source/WebKit/UIProcess/API/glib/webkit.h.in
@@ -90,7 +90,9 @@
 #endif
 #if PLATFORM(GTK)
 #include <@API_INCLUDE_PREFIX@/WebKitPointerLockPermissionRequest.h>
+#if !ENABLE(2022_GLIB_API)
 #include <@API_INCLUDE_PREFIX@/WebKitPrintCustomWidget.h>
+#endif
 #include <@API_INCLUDE_PREFIX@/WebKitPrintOperation.h>
 #endif
 #include <@API_INCLUDE_PREFIX@/WebKitResponsePolicyDecision.h>
